### PR TITLE
#AAQ-329 Improve MKDocs

### PR DESCRIPTION
**Pull request by Tanmay-97**
Reviewer: @sidravi1 

Estimate: 15m

---

## Ticket
Nicer homepage for MKDocs site with feature cards and AAQ logo 

Fixes: [AAQ-329](https://idinsight.atlassian.net/browse/AAQ-329)

## Description, Motivation and Context
The roadmap page and welcome page had a lot of overlap. Now, the welcome page just highlights AAQ's USPs and uses the logo designed by Lenny.
See this ->

https://github.com/IDinsight/aaq-core/assets/63959461/c44a106c-7e19-412c-93c3-06daf4a07135

## How has this been tested?
Locally

## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [ ] I have updated the automated tests (if applicable)
- [ ] I have updated the requirements (if applicable)
- [ ] I have updated the README file (if applicable)
- [ ] I have updated affected documentation (if applicable)

[AAQ-329]: https://idinsight.atlassian.net/browse/AAQ-329?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,6 @@
...
-    __            __        ______
-    /""\          /""\      /    " \
-    /    \        /    \    // ____  \
-    /' /\  \      /' /\  \  /  /    )  )
-  //  __'  \    //  __'  \(: (____/ //
- /   /  \\  \  /   /  \\  \\         \
- (___/    \___)(___/    \___)\"____/\__\
-ASK-A-QUESTION
-</pre>
+<p align="center">
```
Comment (sidravi1) [inline README.md:1]: The full logo does not show up in light mode. 

image

File: README.md [hunk 2]
```diff
@@ -17,7 +10,7 @@ ASK-A-QUESTION
 <a href="#funders-and-partners">Funders and Partners</a>
 </p>
 
-**[Ask-a-question](https://idinsight.github.io/aaq-core/) lets your users ask questions and receive answers from your content repository in their preferred language**
+**[Ask A Question](https://idinsight.github.io/aaq-core/)** is a free and open-source tool created to help non-profit organizations, governments in developing nations, and social sector organizations utilize **Large Language Models** for responding to citizen inquiries in their **native languages**.
```
Comment (sidravi1) [inline README.md:13]: pet peeve of mine: let's sat "use" instead of "utilize".

File: docs/index.md [hunk 3]
```diff
@@ -1,54 +1,57 @@
...
+<div style=" text-align:center; font-size: 36px;"  markdown>
+ :material-star: Features :material-star:
+</div>
 
-**:question: LLM-powered search** : Match your questions to content in the database using embeddings from LLMs.
+<div class="grid cards" style="text-align:center" markdown>
 
-**:robot: LLM responses** : Craft a custom reponse to the question using LLM chat and the content in your database
+- <span style="font-size: 30px" class="primary-color">:material-shield:</span><br>
+  **Resposible & Safe AI** <br>
```
Comment (sidravi1) [inline docs/index.md:23]:
```suggestion
  **Responsible & Safe AI** <br>
```

File: docs/index.md [hunk 4]
```diff
@@ -1,54 +1,57 @@
...
 
-**:books: Manage content** : Use the Admin App to add, edit, and delete content in the database
+- <span style="font-size: 36px;" class="primary-color">:material-docker:</span><br>
+  **Scalable & Easy to Deploy** <br>
+  Containerized app that can integrate with outreach systems like WhatsApp
 
-## :construction: Upcoming
+- <span style="font-size: 30px;" class="primary-color">:material-microphone:</span><br>
+  **Voice support** <br>
+  Voicenote based question-answering support
```
Comment (sidravi1) [inline docs/index.md:36]: Maybe "Ask questions and receive answers using voice memos" to make it clear what the support is?

File: docs/index.md [hunk 5]
```diff
@@ -1,54 +1,57 @@
...
 
-**:woman_technologist: Engineering dashboard**: Monitor uptime, response rates, throughput HTTP reponse codes and more
-
-**:woman_office_worker: Content manager dashboard**: See which content is the most sought after, the kinds of questions that receive poor feedback, identify missing content, and more
+---
 
 !!! question "Looking for other features?"
-    If you are a developing country government, NGO or a social sector organisation, we'd love to hear what features you'd
-    like to see. Raise an issue with `[FEATURE REQUEST]` before the title to start the conversation.
+If you are a developing country government, NGO or a social sector organisation, we'd love to hear what features you'd
```
Comment (sidravi1) [inline docs/index.md:53]: I think that space is needed for this blurb to be under the "Looking for other features" block

File: docs/roadmap.md [hunk 6]
```diff
@@ -8,7 +8,7 @@
 
 !!! info "Key <br> :white_check_mark: - Completed <br> :tools: - In active development <br> :construction: - Queued <br>:pencil: - Yet to be scoped"
 
-On The Horizon
+Beyond 2024
```
Comment (sidravi1) [inline docs/roadmap.md:11]: I should have picked this up previously but the items on the roadmap table don't align. E.g.

image

Or rather, it is dependent on screen size. Can you have each as a row  to make it render correctly?

**Review by sidravi1 — APPROVED** [review]
Thanks @Tanmay-97! Looks so much nicer :)

Couple of small things to fix and then you can merge it in.

File: README.md (hunk 1 above)
Comment (Tanmay-97) [reply README.md:1]: Yes this one is an issue. I tried to set a colored background but it was looking ugly. I'll check with Lenny what he suggests

File: README.md (hunk 2 above)
Comment (Tanmay-97) [reply README.md:13]: Done!

File: README.md (hunk 1 above)
Comment (sidravi1) [reply README.md:1]: Is there a way to switch the image based on what mode it's in?

File: docs/roadmap.md (hunk 6 above)
Comment (Tanmay-97) [reply docs/roadmap.md:11]: Fixed!

File: README.md (hunk 1 above)
Comment (Tanmay-97) [reply README.md:1]: Ah yes, found a fix. Will raise a new PR

**Pull request merged**